src/expr/src/scalar/mod.rs

```diff
@@ -2375,6 +2376,10 @@ impl fmt::Display for EvalError {
             EvalError::InvalidRange(e) => e.fmt(f),
             EvalError::InvalidRoleId(msg) => write!(f, "{msg}"),
             EvalError::InvalidPrivileges(msg) => write!(f, "{msg}"),
+            EvalError::WmrRecursionLimitExceeded(max_iters) => {
```

Comment:
nit: The `Wmr` alludes to the concrete syntax which is more likely to change in the future than the abstract syntax. Can we use `LetRecLimitExceeded` instead?

Reply:
Right, fixed